pkg/systemd/generate/containers.go

```diff
@@ -282,6 +282,22 @@ func setContainerNameForTemplate(startCommand []string, info *containerInfo) ([]
 	return startCommand, nil
 }
 
+func formatOptions(options []string) string {
+	var formatted strings.Builder
+	for _, o := range options {
+		if strings.HasPrefix(o, "-") {
+			formatted.WriteString(" \\\n\t" + o)
+		} else {
+			if formatted.Len() == 0 {
+				formatted.WriteString(o)
+			} else {
+				formatted.WriteString(" " + o)
+			}
```

Comment:
we can remove this branch here if we loop over `options[1:]` and add the first argument before the loop. I think it is safe to assume that we are always called with at least one argument.